Update documentation of rc and nightly switches (#5119)
abpframework/abp: docs/en/CLI.md

* **`add-package`**: Adds an ABP package to a project.
* **`add-module`**: Adds a [multi-package application module](https://docs.abp.io/en/abp/latest/Modules/Index) to a solution.
* **`generate-proxy`**: Generates client side proxies to use HTTP API endpoints on the server.

> * **`switch-to-preview`**: Switches to the latest preview version of the ABP related packages on a solution.

Reviewer: ```suggestion
* **`switch-to-preview`**: Switches to the latest preview version of the ABP Framework.
```

  * **`console`**: [Console template](Startup-Templates/Console.md).
* `--output-folder` or `-o`: Specifies the output folder. Default value is the current directory.
* `--version` or `-v`: Specifies the ABP & template version. It can be a [release tag](https://github.com/abpframework/abp/releases) or a [branch name](https://github.com/abpframework/abp/branches). Uses the latest release if not specified. Most of the times, you will want to use the latest version.

> * `--preview`: Use latest preview version (Only if `--version` is not specified and there is at least one pre-release after latest stable version).

Reviewer: Remove "(Only if `--version` is not specified and there is at least one pre-release after latest stable version)"

### switch-to-preview

> You can use this command to switch your project to latest preview version of the ABP framework packages.

Reviewer: ```suggestion
You can use this command to switch your project to latest preview version of the ABP framework.
```
